### DISCUSSION TITLE
How to specify the certificate to use for TLS in IDicomClient

### DISCUSSION THREAD

**Discussion by pdgmdm** [fo-dicom/fo-dicom#1353] [Q&A]
I am using Fellow Oak 5.0.2 and want to create a IDicomClient to connect to a PACS requiring TLS using certificate.

using this method :
[IDicomClient FellowOakDicom.Network.Client.DicomClientFactory.Create(string host, int port, bool useTls, string callingAe, string calledAe)](https://fo-dicom.github.io/dev/v5/api/FellowOakDicom.Network.Client.DicomClientFactory.html#FellowOakDicom_Network_Client_DicomClientFactory_Create_System_String_System_Int32_System_Boolean_System_String_System_String_)

Where can I specify a X509Certificate ?

**Comment by gofal** [comment]
that is not implemented currently. See #903
